src/api/getMonthlyVulns.tsx

```diff
@@ -0,0 +1,49 @@
+import { getTeamsForOrg } from "./getTeamsForOrg";
+import { getReposForTeam } from "./getReposForTeam";
+import { getVulnsFromRepo } from "./getVulnsFromRepo";
+import { formatVulnData } from "../utils/functions";
+
+import { monthVulns } from "../utils/types";
+
+export const getMonthlyVulns = (graphql:any,  orgLogin: string) => {
+  return getMonthVulns(graphql, orgLogin)
+};
+export async function getMonthVulns(graphql:any, orgLogin:string): Promise<any> {
+  let teams = await getTeamsForOrg(graphql, orgLogin)
+  let monthVulns:monthVulns = {
+    critical: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+    high: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+    moderate: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+    low: [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]
+  }
+
+  let seen = new Set<string>();
+  for(let team of teams) {
+    let newRepos = await getReposForTeam(graphql, orgLogin, team.name)
+    for ( let newRepo of newRepos){
+      if(!seen.has(newRepo.ID)) {
+        seen.add(newRepo.ID)
```

Comment:
Did you mean to do `newRepo.id` here instead? It does not look like the `Repository` type has `ID`